src/index.js

```diff
@@ -174,9 +174,12 @@ class Dropzone extends React.Component {
     })
 
     if (!multiple) {
```

Comment:
I'm wondering if this code can be simplified. Also, should we this be the first check and if this doesn't pass, we should return earlier? As far as I understand it `if (!multiple && this.state.draggedFiles.length > 1)` we want to reject the whole drag operation. Correct?

Reply:
that would also be my logical understanding. i can try it out though and change it if necessary.

Reply:
Did you try this out already? Doesn't look like it has been changed.

Reply:
Working on it now, 'if (!multiple && this.state.draggedFiles.length > 1)' doesn't work, but changing it to 'if (!multiple && this.state.draggedFiles.length === 1)' works, since the length has to be exactly 1 to be accepted as 1 file drop and rejecting multiple file drops', gonna push it when i got my head around the jest test cases.

Reply:
taking it back, it doesn't work at all, that's why tests are failing, gonna let it as it is